### PR TITLE
6070 dod

### PR DESCRIPTION
**Pull request by mrinsin**

### CODE REVIEW

File: web-client/integration-tests/docketClerkSealsCaseContactInformation.test.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  petitionsClerkViewsCaseWithSealedContact(test, contactType);
+
+  loginAs(test, 'docketclerk@example.com');
+  contactType = 'contactSecondary';
+  docketClerkSealsContactInformation(test, contactType);
+
+  loginAs(test, 'petitionsclerk@example.com');
+  petitionsClerkViewsCaseWithSealedContact(test, contactType);
+
+  //the next few tests rely on seed data. If these tests are failing, try restarting the api before re-running the tests
```
Comment (rachaelparris) [inline web-client/integration-tests/docketClerkSealsCaseContactInformation.test.js]: I don't love this. Is there any way to create the data needed for this test within this test?

Comment (mrinsin) [reply web-client/integration-tests/docketClerkSealsCaseContactInformation.test.js]: Agreed, I don't like it either :( unfortunately I don't currently see another way to add other filers/petitioners to a case apart from seed data. I am open to suggestions!

File: web-client/integration-tests/docketClerkSealsCaseContactInformation.test.js [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    'Content-Type': 'application/json',
+  },
+  timeout: 2000,
+});
+
+const otherFilersCase = {
+  ...MOCK_CASE,
+  associatedJudge: CHIEF_JUDGE,
+  caseCaption: 'The Fourth Migrated Case',
+  docketNumber: '187-20',
```
Comment (rachaelparris) [inline web-client/integration-tests/docketClerkSealsCaseContactInformation.test.js]: Could you use a different docketNumber here and in the other case, so it doesn't interfere with anything happening in `caseMigrations.test.js`?

**Review by rachaelparris — APPROVED** [review]

File: web-client/integration-tests/docketClerkSealsCaseContactInformation.test.js (hunk 2 above)
Comment (mrinsin) [reply web-client/integration-tests/docketClerkSealsCaseContactInformation.test.js]: Ooh nice catch! updating now!

File: web-client/integration-tests/docketClerkSealsCaseContactInformation.test.js [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+  caseCaption: 'The Fourth Migrated Case',
+  docketNumber: '199-20',
+  otherFilers: [
+    {
+      address1: '42 Lamb Sauce Blvd',
+      city: 'Nashville',
+      contactId: '46f9ecf7-53d4-43d0-b4ac-8dd340faa219',
+      country: 'USA',
+      countryType: COUNTRY_TYPES.DOMESTIC,
+      email: 'gordon@thelambsauce.com',
```
Comment (kkoskelin) [inline web-client/integration-tests/docketClerkSealsCaseContactInformation.test.js]: Gonna have to recommend `thelambsauce.example.com` and `flavortown.example.com` for the email addresses

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**